pages/basket/index.js

```diff
@@ -17,17 +17,19 @@ import { getProductBasedOnId } from "../../src/database/model";
 // }
 
 const basketArray = [
-  { product_id: 1, quantity: 2 },
-  { product_id: 3, quantity: 1 },
-  { product_id: 4, quantity: 1 },
-  { product_id: 5, quantity: 3 },
+  { product_id: 1, quantity: 2, price: 5 },
+  { product_id: 3, quantity: 1, price: 3 },
+  { product_id: 4, quantity: 1, price: 9 },
+  { product_id: 5, quantity: 3, price: 7 },
 ];
 let itemIds = [];
 let quantity = [];
+let price = [];
 
 basketArray.map((items) => {
   itemIds.push(items.product_id);
   quantity.push(items.quantity);
+  price.push(items.price);
```

Comment:
And you don't need this line either. (No need to push the prices to the array :)